### PR TITLE
rules for clig3

### PR DESCRIPTION
**Pull request by charliejade8888**

### CODE REVIEW

File: src/helpers/seed-db/data/actions.js [hunk 1]
```diff
@@ -112,6 +112,11 @@ export const actions = [
     payment: {
       amountPerHectare: 151
     },
-    eligibilityRules: [{ id: 'is-below-moorland-line' }]
+    eligibilityRules: [
+      {
+        id: 'is-below-moorland-line',
```
Comment (BugBareDrums) [inline src/helpers/seed-db/data/actions.js:117]: Is this supposed to be 'total-area-with-exception', with 'is-below-moorland-line' as another rule?

File: src/rules-engine/rules/total-area-with-exception.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+describe('totalAreaWithException', function () {
+  test('should return true if the area applied for matches the remaining area left by the incompatible action', function () {
+    // Arrange
+    const application = {
+      areaAppliedFor: 50,
+      actionCodeAppliedFor: 'CLIG3',
+      landParcel: {
+        area: 100,
+        existingAgreements: [{ area: 50, code: 'CIGL1' }]
```
Comment (BugBareDrums) [inline src/rules-engine/rules/total-area-with-exception.test.js:16]: A very minor tweak here, I tend to make the part to be subtracted something other than half of the total just to reduce the likelihood of false positive outcomes

File: src/rules-engine/rules/total-area-with-exception.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+export function totalAreaWithException(application, config) {
+  const {
+    areaAppliedFor,
+    actionCodeAppliedFor,
+    landParcel: { existingAgreements, area }
+  } = application
+
+  const incompatibleAction = existingAgreements.find(
+    (/** @type {{ code: any; }} */ agreement) =>
```
Comment (BugBareDrums) [inline src/rules-engine/rules/total-area-with-exception.js:9]: code is a string I believe

File: src/helpers/seed-db/data/actions.js (hunk 1 above)
Comment (charliejade8888) [reply src/helpers/seed-db/data/actions.js]: Good spot! Corrected!

File: src/rules-engine/rules/total-area-with-exception.test.js (hunk 2 above)
Comment (charliejade8888) [reply src/rules-engine/rules/total-area-with-exception.test.js]: Corrected 👍

**Review by BugBareDrums — APPROVED** [review]

**Pull request merged**